src/lightning_app/frontend/frontend.py

```diff
@@ -15,21 +15,21 @@ def __init__(self) -> None:
         self.flow: Optional["LightningFlow"] = None
 
     @abstractmethod
-    def start_server(self, host: str, port: int) -> None:
+    def start_server(self, host: str, port: int, root_path: str = "") -> None:
         """Start the process that serves the UI at the given hostname and port number.
 
         Arguments:
             host: The hostname where the UI will be served. This gets determined by the dispatcher (e.g., cloud),
                 but defaults to localhost when running locally.
             port: The port number where the UI will be served. This gets determined by the dispatcher, which by default
                 chooses any free port when running locally.
-
+            root_path: root_path for the server if app in exposed via a proxy at `/<root_path>`
```

Comment:
```suggestion
            root_path: root_path for the server if app in exposed via a proxy at `/<root_path>`
            
```
You need to leave an empty line

Reply:
Can we include an example like `https://domain.com/<root_path>/my_page` to visualize what it means.